modules/Kconfig.tfm

```
@@ -94,6 +94,30 @@ config TFM_BL2_FALSE
 
 endchoice
 
+config TFM_PARTITION_PROTECTED_STORAGE
+	bool "Enable secure partition 'Protected Storage'"
+	default y
+
+config TFM_PARTITION_INTERNAL_TRUSTED_STORAGE
+	bool "Enable secure partition 'Internal Trusted Storage'"
+	default y
+
+config TFM_PARTITION_CRYPTO
+	bool "Enable secure partition 'Crypto'"
+	default y
+
+config TFM_PARTITION_INITIAL_ATTESTATION
+	bool "Enable secure partition 'Initial Attestation'"
```

This depends on crypto right?
```suggestion
	bool "Enable secure partition 'Initial Attestation'"
	depends on TFM_PARTITION_CRYPTO
```

---

I wanted to leave the dependency resolving out of this.

The reason is that it will, at best, be a clone of functionality which _should_ exist in `trusted-firmware-m` repository.
Then we are in the situation where we have to maintain this, possibly confusing the user if the dependencies stated here are out-of-sync with those it `trusted-firmware-m`.

What do you think?

---

That's fine, as long as the errors from the TFM build system are somewhat understandable and can be traced back to these configs.